doc/source/conf.py

```diff
@@ -72,9 +72,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'2.3.1'
+version = u'2.3.2-rc1'
```

Comment:
same question for this line and line 77 as for the package.json

Reply:
I fixed the packjage.json but left the docs untouched. Will either fix those later